assets/differentialabundance_report.Rmd

```diff
@@ -425,13 +427,34 @@ Input was a matrix of `r nrow(assay_data$raw)` `r params$features_type`s for `r
 
 ## Exploratory analysis
 
+```{r, echo=FALSE, results='asis', fig.height=8, eval=(!is.null(params$features_biotype_col) & !is.null(params$features))}
+cat("### Number of differentially expressed genes by gene type")
```

Comment:
This is not plotting differential genes by biotype, as far as I can see. It’s simply plotting the biotype counts for every gene in the normalised counts table. Do we feel that that’s a useful analysis to be providing, or would it make more sense for this to be done in subsequent sections (e.g near the volcanos), for the differential genes in each contrast?

Reply:
Hi .) @Jonathan Manning thanks for the comments: 

- I am not sure how to better plot differential genes by biotype. What should be the the variable I should use instead?

Reply:
So, what you’re doing right now is just plotting the biotypes for all gene in the dataset (except those removed during filtering). That’s going to be broadly reflective of the geome-wide distribution which is probably not all that interesting, and certainly not what you’ve labelled the plot as ("Number of differentially expressed genes by gene type”).

Assuming that you actually want to plot the biotype distributions for differential genes, you need to insert the plot with the differential analysis. You probably want to plot them separately for up- and down- differential genes, which which case you can use genes in the table derived [here](https://github.com/nf-core/differentialabundance/blob/a7522f3517d448a109097c37b26bbe1e1ad7456c/assets/differentialabundance_report.Rmd#L746).

Reply:
Thanks! I'll work on it. 
Could you maybe point me to a plot showing this up&down reg genes by biotype type of plot, so I could have it as a reference? Thank you very much for your help.

Reply:
I just meant separate plots. But a combined ggplot with two column facets, one for up- and one for down- differential genes could be nice actually.